### PR TITLE
add configuration of ReconnectionPolicy, #105

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: src/main/resources/reference.conf [hunk 1]
```diff
@@ -17,11 +17,11 @@ cassandra-journal {
   # FQCN of the cassandra journal plugin
   class = "akka.persistence.cassandra.journal.CassandraJournal"
 
-  # Comma-separated list of contact points in the cluster.
+  # Comma-separated list of contact points in the Cassandra cluster.
```
Comment (ktoso) [inline src/main/resources/reference.conf]: good clarification

File: src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala [hunk 2]
```diff
@@ -81,11 +85,14 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
...
     )
 
+  val reconnectMaxDelay: FiniteDuration = config.getDuration("reconnect-max-delay", TimeUnit.MILLISECONDS).millis
+
   def clusterBuilder(clusterId: String)(implicit ec: ExecutionContext): Future[Cluster.Builder] = {
     lookupContactPoints(clusterId).map { cp =>
       val b = Cluster.builder
         .addContactPointsWithPorts(cp.asJava)
         .withPoolingOptions(poolingOptions)
+        .withReconnectionPolicy(new ExponentialReconnectionPolicy(1000, reconnectMaxDelay.toMillis))
```
Comment (ktoso) [inline src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala]: 1000 is the backoff multiplier?

**Comment by ktoso** [comment]
LGTM

File: src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala (hunk 2 above)
Comment (patriknw) [inline src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala]: no, it's the start delay, in milliseconds. so it's exponential between start and max.

Comment (ktoso) [inline src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala]: Ok, that makes sense. For multiplier I was confused how 1000 would make sense :)
Thanks!

**Comment by patriknw** [comment]
Refs #105

**Pull request merged**